generator.py

```diff
@@ -135,8 +135,8 @@ def parse_args():
         help="Paths in which to look for Gir.toml files",
     )
     parser.add_argument(
-        "--gir-files-directory",
```

Comment:
@GuillaumeGomez Apparently this was not changed as part of the gtk4-rs commit, only gtk-rs. Fwiw I use `git format-patch -1` and `git am` to move patches between repos :)